app/assets/stylesheets/application.scss

```diff
@@ -1,12 +1,112 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 @import "mixins";
+@import 'https://fonts.googleapis.com/css?family=Balthazar';
 
 /* Global */
 
 hr {
   margin-top: 0;
   margin-bottom: 0;
+  border: 0;
+  height: 1px;
+  background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0));
+}
+
+body {
+  background: url('/assets/concrete.png');
+  font-family: 'Balthazar', serif;
+}
+
+.clr {
+  clear: both;
+}
+
+/*FOOTER*/
+
+.Site {
+  display: flex;
+  min-height: 100vh;
+  flex-direction: column;
+}
+
+.Site-content {
```

Comment:
Not a big deal, but FYI this breaks CSS naming conventions. 
s/Site-content/site-content